src/gsy_e/gsy_e_core/area_serializer.py

```diff
@@ -134,7 +132,7 @@ def optional(attr):
         grid_fee_percentage = description.get('grid_fee_percentage', None)
         grid_fee_constant = description.get('grid_fee_constant', None)
         area = Area(name, children, uuid, optional('strategy'), config,
-                    optional('budget_keeper'), grid_fee_percentage=grid_fee_percentage,
+                    grid_fee_percentage=grid_fee_percentage,
```

Comment:
Should we remove `budget_keeper` also from the `Area` class? E.g. https://github.com/gridsingularity/gsy-e/blob/1fb0a9900d63329ee6f387d81b98ac6fb08aa51a/src/gsy_e/models/area/__init__.py#L115
(and other usages)